server/apps/archive/takes_package_service.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8; -*-
```

Comment:
Since packages functionality is growing I think it's better to refactor to a different package.